### DISCUSSION TITLE
integrating bluepad32

### DISCUSSION THREAD

**Discussion by Spekkie** [StuckAtPrototype/Racer#10] [Ideas]
Saw the video, love the project!
Have you considered using the [bluepad32 lib](https://bluepad32.readthedocs.io/en/latest/) for the esp32?
That way you could use an actual switch controller (or any other Bluetooth controller under the sun...)

**Comment by StuckAtPrototype** [comment]
interesting! I did research a tiny bit, you can definitely get it going with a switch controller 👍
